docs/pages/material-ui/api/alert.json

```diff
@@ -17,13 +17,13 @@
       },
       "default": "{}",
       "deprecated": true,
-      "deprecationInfo": "use the <code>slots</code> prop instead. This prop will be removed in v7. <a href=\"/material-ui/migration/migrating-from-deprecated-apis/\">Learn how to migrate from deprecated APIs</a>."
```

Comment:
@DiegoAndai tagging you as you've implemented the deprecation thingy here — I had changed the link copy from "How to migrate" to "Learn how to migrate from deprecated APIs" but then ran `pnpm docs:api` and it reverted back to "How to migrate" 😶 Should edit this string somewhere else as opposed to individual JSON files? I'm assuming they're automatically generated somehow, and that's why this happened.

Reply:
Ah cool, okay! I do feel like just "How to migrate" sounds weird, though, almost like broken English. "Learn how to migrate" would possibly work, too, but it also feels like it's missing the "migrate from what?" answer which is maybe implied, but still. cc @samuelsycamore for further illuminating thoughts here on our language :)